composer.json

```diff
@@ -23,6 +23,7 @@
         "fabpot/php-cs-fixer": "^1.11"
     },
     "conflict": {
+        "doctrine/dbal": "2.7.0",
```

Comment:
`"doctrine/dbal": ">2.7.0",`

Reply:
Release [2.7.1](https://github.com/doctrine/dbal/compare/v2.7.0...v2.7.1#diff-688c20898c96ed1017c40a3f9b12d051R135) should fix the issue. Could you confirm you're experiencing it with that version?

Reply:
OK, I'm sorry. 
I thought that from 2.7.0 is this problem.